docs/observability/tools/OpenTelemetry.md

```diff
@@ -92,7 +92,7 @@ Java OpenTelemetry instrumentation provides another way to integrate with Azure
 
 When configuring this option, the Applications Insights Agent file is added when executing the application. The `applicationinsights.json` configuration file must be also be added as part of the applications artifacts. Pay close attention to the preview section, where the `"openTelemetryApiSupport": true,` property is set to true, enabling the agent to intercept OpenTelemetry telemetry created in the application code pushing it to Azure Monitor.
 
-OpenTelemetry Java Agent instrumentation supports many [libraries and frameworks and application servers](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#supported-libraries-frameworks-application-servers-and-jvms). Application Insights Java Agent [enhances](https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-in-process-agent#auto-instrumentation) this list. 
```

Comment:
GENERAL RECOMMENDATION: remove files having changes due exclusively to auto-formatting/whitespace edits from this PR and submit as part of a separate PR (focused on misc. whitespace/formatting updates) in order to reduce the 'noise' in this PR.

Reply:
This is fixed by latest merge from main branch.